utils/context/AuthProvider.js

```diff
@@ -19,13 +19,28 @@ export function AuthProvider({ children }) {
   const [user, loading, error] = useAuthState(auth);
   const router = useRouter();
 
-  // console.log(user);
-
-  const signup = (email, password, username) => {
-     createUserWithEmailAndPassword(auth, email, password)
-      .then((response) => {
+  const signup = (body) => {
+    createUserWithEmailAndPassword(auth, body.email, body.password)
+      .then(async (response) => {
+        body['uid'] = response.user.uid
+        if (body.account_type === 'Mentor') {
+           await fetch('/api/pages/mentors/index.js', {
+            method: 'POST',
+            headers: {
+              'Content-Type': 'application/json',
+            },
+            body: body,
+          });
+        } else {
+          await fetch('/api/pages/mentees/index.js', {
+            method: 'POST',
+            headers: {
+              'Content-Type': 'application/json',
+            },
+            body: body,
+          });
+        }
         router.push('/app/my-courses')
-        console.log(response)
       })
       .catch((err) => {
         console.warn('Problem with sign up: ', err.message);
```

Comment:
Fine for now but we should probably look into adding in an error page with a redirect for failed sign ups and log ins